pkg/callback/callbacks.go

```diff
@@ -141,11 +141,12 @@ type AuthenticateHandler struct {
 
 type jwtVerifyClaims struct {
 	Sub       string    `json:"sub"`
+	Iss       string    `json:"iss,omitempty"`
 	Aud       string    `json:"aud"`
-	ThingType ThingType `json:"thingType"`
+	ThingType ThingType `json:"thingType,omitempty"`
```

Comment:
👍 
I think adding `omitempty` to more fields in our message structs will result in a noticeable reduction in message size. We have added it to a lot of fields but when I did my last analysis, I noticed a few that have been missed.